src/python/pants/backend/swift/goals/check_test.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
This won't work if Swift isn't on the filesystem. I saw `golang` has a `conftest.py` which checks for Go, however, that seems to be all/nothing for the backend. 

I can use that approach, but that would mean that the `tailor` tests won't run, even though they don't require the Swift compiler.